asserts/model.go

```diff
@@ -432,6 +433,16 @@ type ModelValidationSet struct {
 	Mode ModelValidationSetMode
 }
 
+func (mvs *ModelValidationSet) AtSequence() *AtSequence {
```

Comment:
this needs some simple dedicated unit tests

Reply:
Good point! Added